src/introduction/prerequisites.md

```diff
@@ -52,29 +52,29 @@ environment:
 :name: installing-cwltool-with-pip-and-venv
 :caption: Installing `cwltool` with `pip` and `venv`.
 
-$ python -m venv venv
+$ python3 -m venv venv
 $ source venv/bin/activate
 $ (venv) pip install -U pip setuptools wheel
 $ (venv) pip install cwltool
 ```
 
 ```{note}
-You can find the `cwl-runner` source code [here](https://github.com/common-workflow-language/cwltool/tree/main/cwlref-runner).
+You can find the `cwl-runner` [source code](https://github.com/common-workflow-language/cwltool/tree/main/cwlref-runner) in the `cwltool` repository.
```

Comment:
This entire line should be deleted

Reply:
There was direct ask from Bruno to add this in.  And then it is better to indicated what the link is instead of saying "here".